### DISCUSSION TITLE
PIL in Vs code

### DISCUSSION THREAD

**Discussion by Te0lexx** [Te0lexx/Hubbo#7] [General]
I got a little problem i don't know why but my vs code doesn't recognize my Pillow.
![pip](https://user-images.githubusercontent.com/83425458/137602520-45d22e59-c50c-4079-94a0-46d3adddddea.PNG)
![pil](https://user-images.githubusercontent.com/83425458/137602529-3af4451a-807d-4724-947c-47e2af90938a.PNG)

**Comment by Te0lexx** [comment]
je pense que le problème vient du fait que VS code ne trouve pas le PATH du pillow.

**Comment by agerard57** [comment]
Have you installed PIL ? 
`pip install --upgrade --force-reinstall pillow`

**Reply by Te0lexx** [reply]
It is in the screenshot i made but i will try to install it with your line.

**Reply by Te0lexx** [reply]
![Screenshot (42)](https://user-images.githubusercontent.com/83425458/137603989-58d2ebb7-fbc7-464b-92af-8b156f42e3cd.png)
Even by reinstalling it won't maybe because the PIL auto installs in an different path than my python path ? i don't know.

**Comment by agerard57** [comment]
Try upgrading pip `pip install --upgrade pip`

**Reply by Te0lexx** [reply]
![cf](https://user-images.githubusercontent.com/83425458/137624294-b9e86a6f-bc31-420a-b6d2-ac7bd77c72d6.PNG)
Upgraded pip but it says not on actual Path.

**Comment by ItsPizzaTimez** [comment]
moi je sais pourquoi ca marche pas, faut pas tout importer en tkinter quand tu utilises pillow. Importe que se que tu utilises comme Entry etc...
Ah et je crois j'ai pas rep au probleme mdr. Mais voila xdddddddddd
Ptdr je nique tout avec mon francais
Sa se trouve ta pip au lycée donc leur proxy bloque les installations depuis qqlq par, sa me fait le meme prblm
